StablecoinResearch/Stablecoins/Model/experiments_refined/comparative_model_analysis.py

```
@@ -0,0 +1,341 @@
+                    linewidth=2
+                )
+        
+        plt.xlabel('Pool Depth ($)')
+        plt.ylabel('Minimum Attack Size for 50% Failure (%)')
+        plt.title('Attack Threshold Comparison Across Models')
+        plt.legend()
+        plt.grid(True, alpha=0.3)
+        plt.xscale('log')
+        
+        # Format x-axis
+        plt.gca().xaxis.set_major_formatter(plt.FuncFormatter(lambda x, p: f'${x:,.0f}'))
```

Ruff (0.12.2)

Unused lambda argument: `p`

(ARG005)